### PR TITLE
Soufianej k3s playbook

### PR DESCRIPTION
**Pull request by JOUNAIDSoufiane**
- Removed the ansible_managed flag from k3s template files
- Added a pip task to install the kubernetes python library

### CODE REVIEW

File: roles/k3s/tasks/config-k3s-service.yml [hunk 1]
```diff
@@ -60,3 +60,9 @@
     daemon_reload: yes
     state: started
     enabled: yes
+
+- name: Install Kubernetes Python client
+  become: true
+  pip:
+    name: kubernetes
+    state: present
```
Comment (msherman64) [inline roles/k3s/tasks/config-k3s-service.yml]: Did you verify that this installed client into `/etc/ansible/venv`, such as by deleting the venv, then re-creating with bootstrap-servers prior to running this playbook?

Also, we should pin the version of python-kubernetes installed, or we might get unexpected breakage later

**Review by msherman64 — APPROVED** [review]
LGTM, assuming nits are double-checked

File: roles/k3s/tasks/config-k3s-service.yml (hunk 1 above)
Comment (JOUNAIDSoufiane) [reply roles/k3s/tasks/config-k3s-service.yml]: sure looking into

**Review by codyhammock — APPROVED** [review]
LGTM

**Pull request merged**